local-volume/helm/provisioner/templates/provisioner.yaml

```diff
@@ -69,7 +69,7 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: spec.nodeName
-          - name: MY_NAMESPACE
+          - name: JOB_NAMESPACE
```

Comment:
How about PROVISIONER_NAMESPACE?  Both the provisioner and its jobs should run in the same namespace.

Reply:
From [this discussion](https://github.com/kubernetes-incubator/external-storage/pull/711/files#r181179874), I thought we allow admin to specify arbitrary namespace to run jobs in. 

If we require jobs must be run in provisioner namespace, I think `MY_NAMESPACE` is a good name, because `MY` means `PROVISIONER` here and we already use `MY_NODE_NAME` to specify node provisioner run on.